### PR TITLE
Add python_version to Pipfile

### LINKED ISSUE #19: Add python version to Pipfile

### PR DESCRIPTION
**Pull request by nealedj**
Closes #19

### CODE REVIEW

**Comment by shahi645** [comment]
me and @nealedj tested this yesterday, not a cause of this PR

**Comment by nealedj** [comment]
I'm seeing the same. Tried setting the python_version to 3.4.2 and recreating the virtualenv but the same thing happened.

**Comment by shahi645** [comment]
@ajmaddaford do you know the cause of this issue?

**Comment by ajmaddaford** [comment]
When we had this issue on survey runner it was due to an incompatibility between the pylint-mcabe and astroid modules - ONSdigital/eq-survey-runner#1418. It was fixed by pinning astroid to version 1.5.3.

**Comment by nealedj** [comment]
Just tested this again (recreated virtualenv) and the errors aren't appearing during linting anymore.

**Comment by nealedj** [comment]
@jonnyshaw89 would you mind testing this again?

**Comment by jonnyshaw89** [comment]
@nealedj pet hate of mine is squashing before PR is approved. It makes it hard when re-reviewing to see what has changed.

**Comment by nealedj** [comment]
Acknowledged :) Thanks for letting me know.

**Comment by nealedj** [comment]
See #24 to fix linting errors.

**Comment by jonnyshaw89** [comment]
@nealedj I think we need to fix the Linting errors here otherwise we are knowingly merging "broken" code

**Comment by nealedj** [comment]
@jonnyshaw89 Yep fair enough. But it should still be a separate issue as the changes in this PR didn't cause the issue.

**Comment by nealedj** [comment]
Now fixes #24 also.

File: Pipfile [hunk 1]
```diff
@@ -17,5 +17,11 @@ structlog = "*"
...
 "flake8" = "==3.4.1"
 pylint = "==1.7.4"
+astroid = "==1.5.3"
 pylint-mccabe = "==0.1.3"
 pylint-quotes = "==0.1.6"
+
+
+[requires]
+
+python_version = "3.6.3"
```
Comment (jonnyshaw89) [inline Pipfile]: Should we just be specifying the major.minor here and let the lock specify the patch version?

File: .python-version [hunk 2]
```diff
@@ -1 +0,0 @@
-3.6.3
```
Comment (jonnyshaw89) [inline .python-version]: @nealedj I can get it to work if this file is back in.
Without this pipenv doesn't know which version of python to use. which to me is quite stupid

Comment (nealedj) [reply .python-version]: Hmm, go figure. I've added it back again.

**Comment by dwyeradam** [comment]
@nealedj  Can be squashed and rebased now.

**Pull request merged**